lib/endpoints/class-wp-rest-controller.php

```diff
@@ -250,7 +250,6 @@ public function get_collection_params() {
 				'default'            => 10,
 				'minimum'            => 1,
 				'maximum'            => 100,
-				'sanitize_callback'  => 'absint',
```

Comment:
Is this problem should solved on WordPress core?
It looks that validation callback fires after sanitization callback.